app/src/main/java/nl/tudelft/pixelperfect/client/EventsMessage.java

```diff
@@ -0,0 +1,38 @@
+package nl.tudelft.pixelperfect.client;
+
+import com.jme3.network.AbstractMessage;
+import com.jme3.network.serializing.Serializable;
+
+/**
+ * The class that is sent by the server when it shares an event.
+ *
+ * @author Dmitry Malarev.
+ */
+@Serializable
+public class EventsMessage extends AbstractMessage{
+    private String event;
```

Comment:
I am talking about this message class itself: it will still be serializable if you add some primitive data fields to it like an integer for the type, a long for the duration and a double for the damage. If we were only using 1 String field we could just use one single class for all messages, which makes no sense at all.